wolfcrypt/src/rsa.c

```diff
@@ -1143,21 +1153,21 @@ static int RsaPad_OAEP(const byte* input, word32 inputLen, byte* pkcsBlock,
         return ret;
     }
 
+#ifdef WOLFSSL_SMALL_STACK
```

Comment:
Can we add `WOLFSSL_NO_MALLOC`?
`#if defined(WOLFSSL_SMALL_STACK) && !defined(WOLFSSL_NO_MALLOC)`